src/widgets/slideshow/SlideshowTile.tsx

```diff
@@ -0,0 +1,94 @@
+import { Center, Group, Stack, Title } from '@mantine/core';
+import { IconPhoto, IconPhotoX } from '@tabler/icons';
+import { useTranslation } from 'next-i18next';
+import React from 'react';
+import { Slide, Fade, Zoom } from 'react-slideshow-image';
+import { defineWidget } from '../helper';
+import { IWidget } from '../widgets';
+import 'react-slideshow-image/dist/styles.css';
+
+const definition = defineWidget({
+  id: 'slideshow',
+  icon: IconPhoto,
+  options: {
+    FeedUrl: {
+      type: 'text',
+      defaultValue: '',
+    },
+    Effect: {
+      type: 'select',
+      defaultValue: 'Fade',
+      data: [
+        { label: 'Slide', value: 'Slide' },
+        { label: 'Fade', value: 'Fade' },
+        { label: 'Zoom', value: 'Zoom scale={0.4}' },
+      ],
+    },
+    Duration: {
+      type: 'number',
+      defaultValue: 5000,
+    },
+  },
+  gridstack: {
+    minWidth: 3,
+    minHeight: 3,
+    maxWidth: 12,
+    maxHeight: 12,
+  },
+  component: SlideshowWidget,
+});
+
+export type SlideshowWidget = IWidget<(typeof definition)['id'], typeof definition>;
+
+interface SlideshowWidgetProps {
+  widget: SlideshowWidget;
+}
+
+function SlideshowWidget({ widget }: SlideshowWidgetProps) {
+  const { t } = useTranslation('modules/slideshow');
+  const divStyle = {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    backgroundSize: 'cover',
+    height: '400px',
+  };
+  const Images = [
```

Comment:
I assume this is for testing? You should remove it and read from the options. This probably depends on #704